scp/scpProcessDestroyer.cpp

```diff
@@ -1,103 +1,144 @@
 /*
-* This source file is part of an OSTIS project. For the latest info, see http://ostis.net
-* Distributed under the MIT License
-* (See accompanying file COPYING.MIT or copy at http://opensource.org/licenses/MIT)
-*/
+ * This source file is part of an OSTIS project. For the latest info, see http://ostis.net
+ * Distributed under the MIT License
+ * (See accompanying file COPYING.MIT or copy at http://opensource.org/licenses/MIT)
+ */
 
+#include "scpProcessDestroyer.hpp"
 #include "scpKeynodes.hpp"
 #include "scpUtils.hpp"
-#include "scpProcessDestroyer.hpp"
-#include "sc-memory/sc_memory.hpp"
+
 #include <iostream>
 
 namespace scp
 {
-ScAddr ASCPProcessDestroyer::msAgentKeynode;
-
-SC_AGENT_IMPLEMENTATION(ASCPProcessDestroyer)
+ScResult ASCPProcessDestroyer::DoProgram(
```

Comment:
Could you write docs (algorithms) for agents that were significantly changed by you?